src/stores/index.js

```diff
@@ -1020,6 +1031,76 @@ class Store {
     })
   }
 
+  getLeaderBoard = (_payload) => {
+    
+    const account = store.getStore('account')
+    const web3 = new Web3(store.getStore('web3context').library.provider);
+    const memes = this.getMemes();
+
+    async.map(memes, (meme, callback) => {
+        this._getLeaderBoardData(web3, meme, callback);
+    }, async(err, leaderboardData) => {
+      if(err) {
+        return emitter.emit(ERROR, err);
+      }
+      console.log(leaderboardData);
+      leaderboardData.sort(function(a, b) {
+        return a.score - b.score; 
+      })
+
+      leaderboardData.reverse();
+
+      
+      store.setStore({ leaderboard: leaderboardData })
+      emitter.emit(GET_LEADERBOARD_RETURNED);
+      console.log("Rank  Post ID          Poster                               Votes For   Votes Against          Adj. Factor   Score")
+      for(var i = 0; i < leaderboardData.length; i++) {
+        console.log(
+          i+1+"     ",
+          leaderboardData[i].id+"    ",
+          leaderboardData[i].poster+ "  ",
+          leaderboardData[i].votesFor+ "  ",
+          leaderboardData[i].votesAgainst+ "  ",
+          leaderboardData[i].voters+ "  ",
+          leaderboardData[i].factor+ "  ",
+          leaderboardData[i].score
+        )
+      }
```

Comment:
yeah please do remove this

Reply:
It was just there for the Shaftoe to verify the data

Reply:
same as above